src/utils/fileUtils.js

```diff
@@ -22,6 +22,23 @@
 import camelcase from 'camelcase'
 import { isNumber } from './numberUtil'
 
+/**
+ * Get an url encoded path
+ *
+ * @param {String} path the full path
+ * @returns {string} url encoded file path
+ */
+const encodeFilePath = function(path) {
```

Comment:
Don't think this is necessary to add back in but we can just do that in another PR :).